ext/hash/hash.stub.php

```diff
@@ -39,11 +39,11 @@ function mhash_get_block_size(int $algo): int|false {}
 
 function mhash_get_hash_name(int $algo): string|false {}
 
-function mhash_keygen_s2k(int $algo, string $password, string $salt, int $length): string|false {}
+function mhash_keygen_s2k(int $algo, string $password, string $salt, int $length, array $args = []): string|false {}
 
 function mhash_count(): int {}
 
-function mhash(int $algo, string $data, ?string $key = null): string|false {}
+function mhash(int $algo, string $data, ?string $key = null, array $args = []): string|false {}
```

Comment:
mhash_* functions are legacy APIs provided for backwards compatibility (really should be removed). Personally, I wouldn't change them. But it's also not a problem either.

Reply:
Makes sense. Also it reduces the patch which is good, too. I fixed this, removed for `mhash_keygen_s2k` as well.

Thanks